app/components/global_threads/thread_footer/index.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {bindActionCreators, Dispatch} from 'redux';
+import {connect} from 'react-redux';
+import {injectIntl} from 'react-intl';
+
+import {setThreadFollow} from '@mm-redux/actions/threads';
+import {getCurrentUserId} from '@mm-redux/selectors/entities/common';
+import {getCurrentTeamId} from '@mm-redux/selectors/entities/teams';
+import type {GlobalState} from '@mm-redux/types/store';
+
+import ThreadFooter, {StateProps, DispatchProps, OwnProps} from './thread_footer';
+
+function mapStateToProps(state: GlobalState) {
+    return {
+        currentTeamId: getCurrentTeamId(state),
```

Comment:
Resolving as now this component seems not to mount anywhere other than the global threads and the channel screen

Reply:
This one is fixed in the latest push. We are getting team id in action instead of sending from component as you have suggested. I think it makes sense and also keeps the code clean. I might try doing the same for other things. Thanks.